### DISCUSSION TITLE
Questions about the choice of k mesh

### DISCUSSION THREAD

**Discussion by Lamed-git** [ttadano/alamode#162] [Q&A]
Dear Dr. Tadano,
I want to know how to determine the parameter KMESH_INTERPOLATE in SCPH calculation, I read your PPT handout, "The q points sampled by KMESH_INTERPOLATE should be commensurate with the supercell size (of harmonic IFCs)" is mentioned aboveion. But I'm not sure what it means. For example, a single cell of size 12 12 4 angstroms is expanded to 12 12 12 angstroms for harmonic calculation, so should the KMESH_INTERPOLATE be 2 2 2 or 1 1 3? In addition, when calculating the thermal conductivity of the lattice, whether the "&kpoint"-field should be the same as the DOS calculation, that is, related to the reciprocal crystal vector, such as the 12 12 4 angstroms of the single cell k mesh is 3 3 9 or 5 5 5?
Looking for your reply!

**Comment by ttadano** [comment]
If you use a 1x1x3 supercell for the harmonic IFC calculation, the KMESH_INTERPOLATE should be  `KMESH_INTERPOLATE = 1 1 3`. `KMESH_INTERPOLATE = 1 1 1` is another possible choice, but the former choice should be more accurate.

For DOS and kL calculations, you do not need to use the same k-point mesh.

**Reply by Lamed-git** [reply]
Thank you for your reply. Perhaps I did not express the second question clearly. I wonder whether isotropic or anisotropic k mesh should be used to calculate the lattice thermal conductivity for anisotropic crystals. In addition, I wonder whether the Coherent component of lattice thermal conductivity should be considered on the basis of SCPH. Generally speaking, the lattice thermal conductivity calculated by BTE+Coherent component is smaller than that by SCPH+Coherent component.
